### ISSUE TITLE
White .Tif saved

### ISSUE THREAD

**Issue by EllenMil** [HenriquesLab/ZeroCostDL4Mic#24]
Hi,
I'd like to first thank everyone who has helped creating this wonderful tool. This summer, I'm working with a lab that is trying to reduce noise in their fluorescent microscopy images (using MacOS). 

To learn more about how the notebooks work, I used the provided sample datasets with both the Noise2Void (2D) and CARE (2D) notebooks. I've come across two different trouble spots. I've tried to resolve the problem, but would like you team's opinion on it. 

First problem that I encounter is when I'm setting training parameters (in cell 3.1). I've tried to set the patch size to 32 and 16. But I receive the a value error when I'm training the network (cell 4.2). 
See screenshots below. It seems that I can train the network only when the patch size is set to 64. 
<img width="938" alt="Screen Shot 2020-06-05 at 12 51 29 PM" src="https://user-images.githubusercontent.com/66487910/83908602-03f55c80-a72d-11ea-84e2-5e06da8290b5.png">
<img width="1061" alt="Screen Shot 2020-06-05 at 12 53 30 PM" src="https://user-images.githubusercontent.com/66487910/83908617-0a83d400-a72d-11ea-9b5b-e5b57ea7fa65.png">
<img width="1059" alt="Screen Shot 2020-06-05 at 12 54 24 PM" src="https://user-images.githubusercontent.com/66487910/83908649-18395980-a72d-11ea-8ebd-e4d3ffa3fcd0.png">

The other problem that I encounter is when I'm trying to save the prediction outputs. When I'm assessing the outputs (in cell 6.2), the colab notebook displays two beautiful before and after images. However, the predicted .tif files in my drive and on my computer show up as completely white. See screenshots below. 
<img width="1062" alt="Screen Shot 2020-06-05 at 12 57 44 PM" src="https://user-images.githubusercontent.com/66487910/83908982-9269de00-a72d-11ea-96a4-c35be0e75208.png">
<img width="749" alt="Screen Shot 2020-06-05 at 12 58 51 PM" src="https://user-images.githubusercontent.com/66487910/83909006-9c8bdc80-a72d-11ea-8fd0-be3f0662b588.png">

I would appreciate hearing back from this amazing team. 
Once again, thank you
Ellen

**Comment by Romain-Laine** [comment]
Hi Ellen, 
Thanks for your feedback. It's always useful for us to get info on how people get on with our notebooks. 
Regarding the patch_size issue, I have not seen that problem before and will need to look into that. 

Just to confirm, you're using the latest version, right? With our provided example dataset, right? Is the issue definitely reproducible (or only sometimes)?

Regarding your issue of white images, have you tried opening them in Fiji/ImageJ and see if they look ok in there (after doing an auto-contrast)?

Cheers,

Romain

**Comment by EllenMil** [comment]
Romain,
Thanks for getting back to me!
I was using Photoshop automate to auto-contrast all the images. But thank you for suggesting the Fiji/ImageJ software. It solved the white image problem!

I just checked if I'm using the latest version of the Noise2Void and CARE notebooks, so I'm not sure why changing the batch size gives that value error every time I run the code chunk.  I tried changing the number of epochs and the code runs just fine. I'm not sure why it doesn't like me changing the number of patches. 
Have a good week,
Ellen
